cmd/opdoc/opdoc.go

```diff
@@ -236,44 +236,40 @@ func opsToMarkdown(out io.Writer) (err error) {
 
 // OpRecord is a consolidated record of things about an Op
 type OpRecord struct {
-	Opcode  byte
+	Opcode  byte `json:",omitempty"`
 	Name    string
 	Args    string `json:",omitempty"`
 	Returns string `json:",omitempty"`
 	Size    int
 
-	ArgEnum      []string `json:",omitempty"`
-	ArgEnumTypes string   `json:",omitempty"`
+	ArgEnum string `json:",omitempty"`
 
-	Doc               string
-	DocExtra          string `json:",omitempty"`
-	ImmediateNote     string `json:",omitempty"`
-	IntroducedVersion uint64
-	Groups            []string
+	Doc               string   `json:",omitempty"`
+	DocExtra          string   `json:",omitempty"`
+	ImmediateNote     string   `json:",omitempty"`
+	IntroducedVersion uint64   `json:",omitempty"`
+	Groups            []string `json:",omitempty"`
+}
+
+type Keyword struct {
+	Name      string
+	Type      string
+	ValueEnum []string `json:",omitempty"`
```

Comment:
Nothing written to use this yet, going back and forth on whether we should embed the TxnType/OnCompletes here or at the same level as other keyword groups.